src/lib/IRTransponder/Transponder.h

```diff
@@ -0,0 +1,16 @@
+//
+// Authors: 
+// * Dominic Clifton (hydra)
+//
+
+#pragma once
+
+class TransponderSystem {
+public:
+    virtual ~TransponderSystem() {};
+    virtual bool init() = 0;
+    virtual void deinit() = 0;
+    virtual void startTransmission() = 0;
+    virtual bool isInitialised() = 0;
+};
```

Comment:
Since this is public interface for transponders it should have documentation similar to `handset.h`